src/Service.Tests/GraphQLBuilder/MsSqlNestedMutationBuilderTests.cs

```diff
@@ -6,13 +6,13 @@
 
 namespace Azure.DataApiBuilder.Service.Tests.GraphQLBuilder
 {
-    [TestClass]
+    [TestClass, TestCategory(TestCategory.MSSQL)]
     public class MsSqlNestedMutationBuilderTests : NestedMutationBuilderTests
     {
         [ClassInitialize]
         public static async Task SetupAsync(TestContext context)
         {
-            databaseEngine = "MsSql";
+            databaseEngine = TestCategory.MSSQL;
```

Comment:
Yes. In attribute, it needs to be defined so that the pipelines categorize it as an mssql test. For the test itself (local/remote), databaseEngine needs to be set.